### ISSUE TITLE
Parker parsing list as single element

### ISSUE THREAD

**Issue by yuriescl** [shamblett/xml2json#41]
How to reproduce:
```
var transformer = Xml2Json();
transformer.parse('<root><a>1</a><a>2</a></root>');
print(transformer.toParker());
```
Output:
```
{"root": {"a": "2"}}
```
Expected:
```
{"root": {"a": ["1", "2"]}}
```

Version: `xml2json: ^5.0.0`

`flutter doctor -v`:
```
[✓] Flutter (Channel stable, 2.2.2, on Linux, locale en_US.UTF-8)
    • Flutter version 2.2.2 at /home/yuri/opt/flutter
    • Framework revision d79295af24 (2 days ago), 2021-06-11 08:56:01 -0700
    • Engine revision 91c9fc8fe0
    • Dart version 2.13.3

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /home/yuri/Android/Sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /home/yuri/Android/Sdk
    • Java binary at: /home/yuri/opt/android-studio-ide-202.7351085-linux/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+0-b944-P17168821)
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/linux#android-setup for more details.

[✗] Chrome - develop for the web (Cannot find Chrome executable at google-chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Android Studio (version 4.2)
    • Android Studio at /home/yuri/opt/android-studio-ide-202.7351085-linux
    • Flutter plugin version 57.0.2
    • Dart plugin version 202.8488
    • Java version OpenJDK Runtime Environment (build 11.0.8+0-b944-P17168821)

[✓] Connected device (1 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86 • Android 7.0 (API 24) (emulator)
```

**Comment by yuriescl** [comment]
That section of the code was changed since the version I tested (`^5.0.0`), so I tested using the newest version.

Tested using `xml2json: ^5.3.0`:
```
var transformer = Xml2Json();
transformer.parse('<root><a>1</a><a>2</a><a>3</a><a>4</a></root>');
print(transformer.toParker());
```
Output:
```
{"root": {"a": ["1", "2", {}, {}]}}
```

**Comment by shamblett** [comment]
You should be using parker with attributes -
```
oid main() {
  test('List as single element', () {
    var transformer = Xml2Json();
    transformer.parse('<root><a>1</a><a>2</a><a>3</a><a>4</a></root>');
    final output = transformer.toParkerWithAttrs();
    print(output);
    expect(output, '{"root": {"a": ["1", "2", "3", "4"]}}');
  });
```

**Comment by yuriescl** [comment]
> You should be using parker with attributes -
> 
> ```
> oid main() {
>   test('List as single element', () {
>     var transformer = Xml2Json();
>     transformer.parse('<root><a>1</a><a>2</a><a>3</a><a>4</a></root>');
>     final output = transformer.toParkerWithAttrs();
>     print(output);
>     expect(output, '{"root": {"a": ["1", "2", "3", "4"]}}');
>   });
> ```

Why should I be using Parker with attributes if I don't have/want attributes?
Also, there's an issue with the Parker anyways and the fix is in the pull request.

**Comment by yuriescl** [comment]
Fixed https://github.com/shamblett/xml2json/pull/42

**Comment by shamblett** [comment]
Your pull request breaks no existing unit tests for Parker and adds the functionality you want, merged it, re published the package at version 5.3.1
